.github/workflows/perl-critic.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: 'Perl critic'
+
+on: [push, pull_request]
+
+jobs:
+  perl-critic-checks:
+    runs-on: ubuntu-latest
+    name: "Perlcritic"
+    container:
+      image: perldocker/perl-tester
+    steps:
+      - uses: actions/checkout@v4
+      - run: make test-critic
```

Comment:
```
Policy "Perl::Critic::Policy::HashKeyQuotes" is not installed.
Policy "Perl::Critic::Policy::RedundantStrictWarning" is not installed.
```

I see those in CI 🤔

Reply:
Thanks, I missed that. Makefile needs to run tools/perlcritic instead. pushed. Now we should see failures and I'm currently trying to fix them